apps/src/templates/sharing.html.ejs

```diff
@@ -41,7 +41,7 @@
       </button>
     <% } else if (options.saveToProjectGallery && !options.disableSaveToGallery) { %>
       <button id="save-to-project-gallery-button">
-        <%= msg.save() %>
+        <%= msg.addToProjects() %>
```

Comment:
I dont think I realized we still had ejs files :cry:

Reply:
Fortunately this dialog is slated for redesign very soon.